src/main/java/com/spotinst/sdkjava/model/requests/elastigroup/aws/ElastigroupImportASGRequest.java

```diff
@@ -0,0 +1,54 @@
+package com.spotinst.sdkjava.model.requests.elastigroup.aws;
+
+import com.spotinst.sdkjava.client.rest.JsonMapper;
+import com.spotinst.sdkjava.model.ElastigroupConverter;
+import com.spotinst.sdkjava.model.api.elastigroup.aws.ApiElastigroupImportASG;
+import com.spotinst.sdkjava.model.bl.elastigroup.aws.ElastigroupImportASG;
+
+public class ElastigroupImportASGRequest {
```

Comment:
This class is not needed. Parameters can be passed in the call itself.

Reply:
I have created request class as per the standard which we are using in our api automation.

Reply:
As we already discuss on this.